ChangeLog.md

```diff
@@ -17,18 +17,22 @@ See docs/process.md for how version tagging works.
 
 Current Trunk
 -------------
+- Upstream Clang now defaults to `-fno-common`. See
+  https://reviews.llvm.org/rG2c36c23f347
```

Comment:
I think we already default to "-fno-common" for wasm so this is probably not worth mentioning.   Checking now..

Reply:
Indeed: https://github.com/llvm/llvm-project/blob/6e29846b29d2bcaa8a7a3d869f24f242bd93d272/llvm/lib/MC/MCWasmStreamer.cpp#L139

Reply:
Thanks, removing that part.